reportportal/templates/uat-deployment.yaml

```diff
@@ -66,6 +66,14 @@ spec:
         - name: RP_BINARYSTORE_MINIO_SECRETKEY
           value: "{{ .Values.minio.secretkey }}"
         {{ end }}
+        {{ if .Values.minio.bucketPrefix}}
+        - name: RP_BINARYSTORE_MINIO_BUCKETPREFIX
+          value: "{{ .Values.minio.bucketPrefix }}"
+        {{ end }}
+        {{ if .Values.minio.defaultBucketName}}
+        - name: RP_BINARYSTORE_MINIO_DEFAULTBUCKETNAME
+          value: "{{ .Values.minio.defaultBucketName }}"
+        {{ end }}
```

Comment:
Add the RP_INTEGRATION_SALT_PATH also for uat, as it's used by both api and uat.

Reply:
Done.